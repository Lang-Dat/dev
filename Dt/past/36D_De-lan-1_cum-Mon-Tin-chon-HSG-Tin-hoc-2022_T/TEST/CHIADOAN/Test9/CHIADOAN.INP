90
35037 29053 6083 58007 16223 47867 60255 3835 2400 61690 51017 13073 60538 3552 24414 39676 2254 61836 55745 8345 14858 49232 45706 18384 34475 29615 38725 25365 53385 10705 58198 5892 12200 51890 34763 29327 25610 38480 7846 56244 36496 27594 8294 55796 46631 17459 59946 4144 36459 27631 42625 21465 33097 30993 36778 27312 1678 62412 16676 47414 33398 30692 27329 36761 51790 12300 34835 29255 48640 15450 23546 40544 4158 59932 38564 25526 11076 53014 3814 60276 30397 33693 4617 59473 43318 20772 1812 62278 31000 33090 9766 54324 30646 33444 19200 44890 37249 26841 4482 59608 26277 37813 61593 2497 3639 60451 35525 28565 40619 23471 17721 46369 7419 56671 47735 16355 3841 60249 20366 43724 47584 16506 21453 42637 53728 10362 4638 59452 56817 7273 43545 20545 9504 54586 18453 45637 54248 9842 19908 44182 14130 49960 62414 1676 16904 47186 1325 62765 8504 55586 46625 17465 17646 46444 17160 46930 45423 18667 38360 25730 28486 35604 33909 30181 7493 56597 33705 30385 25696 38394 61639 2451 37235 26855 12593 51497 49263 14827 13609 50481 