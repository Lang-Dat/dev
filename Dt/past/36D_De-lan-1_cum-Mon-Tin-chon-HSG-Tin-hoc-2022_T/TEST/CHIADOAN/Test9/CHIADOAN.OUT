45 64090
35037 29053 
6083 58007 
16223 47867 
60255 3835 
2400 61690 
51017 13073 
60538 3552 
24414 39676 
2254 61836 
55745 8345 
14858 49232 
45706 18384 
34475 29615 
38725 25365 
53385 10705 
58198 5892 
12200 51890 
34763 29327 
25610 38480 
7846 56244 
36496 27594 
8294 55796 
46631 17459 
59946 4144 
36459 27631 
42625 21465 
33097 30993 
36778 27312 
1678 62412 
16676 47414 
33398 30692 
27329 36761 
51790 12300 
34835 29255 
48640 15450 
23546 40544 
4158 59932 
38564 25526 
11076 53014 
3814 60276 
30397 33693 
4617 59473 
43318 20772 
1812 62278 
31000 33090 
